docs/guide/writing-migrations.md

```diff
@@ -249,7 +244,7 @@ module.exports = {
               await self.apos.doc.db.updateOne({
                 _id: doc._id
               }, {
-                $unset: { `${dotPath.border}`: '' }
+                $unset: { `${dotPath}.border`: '' }
```

Comment:
... Huh. I see that your empty string "value" is consistent with how mongodb examples handle this, I guess they like a natural "falsy"/"empty" value for whatever the type was, even though the value does not matter at all with `$unset`. 👍